bluepyemodel/icselector/icselector.py

```diff
@@ -0,0 +1,379 @@
+""" ICSELECTOR
+Select a set of NEURON mechanisms, parameters and bounds from corresponding
+genes. Gene names can be selected from a file mapping genes to different
+ME-T types. Corresponding channels are selected from a file mapping channels
+and parameters to genes.
+
+## Usage
+Select from a gene-mapping file:
+    $ python icselector.py
```

Comment:
good idea @AurelienJaquier .  @vandorp I also need to make some changes in the code because we are not supporting special characters in the gene map anymore. I think it will be easier if I create another PR after this one. If you'd like, I can take care of that as well

Reply:
Command line example moved to examples folder in [a52a298](https://github.com/BlueBrain/BluePyEModel/pull/144/commits/a52a298d15735b4069c7b4b63df6c5b354383bf0)